# Exercises - Day 1

- Lists
- Tuples
- Sequence functions
- Dictionaries 
- Sets
- Logic statements
- If statements
- For loops
- While loops
- List comprehensions


### Exercise 

Use the list: [1.56680276, 7.03735315, 5.02030225, 6.36853766, 1.07804003, 0.37483712, 8.85174971, 0.30987579, 3.32970315, 0.01512192, 6.79334122, 0.42385231, 3.85455554, 0.46288301, 3.97774927, 7.00411013, 0.96858288, 4.22386816, 4.26947196, 2.7484385 ]

In [10]:
import numpy as np 
np.random.uniform(0,10,20)

array([1.56680276, 7.03735315, 5.02030225, 6.36853766, 1.07804003,
       0.37483712, 8.85174971, 0.30987579, 3.32970315, 0.01512192,
       6.79334122, 0.42385231, 3.85455554, 0.46288301, 3.97774927,
       7.00411013, 0.96858288, 4.22386816, 4.26947196, 2.7484385 ])

### Exercise 1.1
Write a program that prints all the numbers from 0 to 6 exept 3 and 6:
- Using a for loop and if statements.
- Using a for loop, an if statement and the continue statement.

Expected Result : 0 1 2 4 5

### Exercise 1.2
Write a Python program which takes two digits m (row) and n (column) as input and generates a two-dimensional array. <br>
The element value in the i-th row and j-th column of the array should be i*j.

Test Data : Rows = 3, Columns = 4 <br>
Expected Result : \[\[0, 0, 0, 0\], \[0, 1, 2, 3\], \[0, 2, 4, 6\]\]


### Ecxercise 1.3
Write a Python program that creates a list of the Fibonacci series between 0 to 50. <br> 
The Fibonacci Sequence is the series of numbers : <br>
>0, 1, 1, 2, 3, 5, 8, 13, 21, ...., <br>

where the next number is the sum of the two last numbers.

Expected Output : \[0, 1, 1, 2, 3, 5, 8, 13, 21, 34\]

### Exercise 1.4
Acces the value of key "history" from sampleDict.

In [ ]:
sampleDict = { 
   "class":{ 
      "student":{ 
         "name":"Mike",
         "marks":{ 
            "physics":70,
            "history":80
         }
      }
   }
}

### Exercise 1.5
Create two dictionary called "prices" and "stock". Add the following fruits to each dictionary with a suitable value: pear, apple, orange, banana and lemon.

Print the total value of the stock using a for loop.

### Exercise 1.6 
Use list comprehensions to make a list of numbers between 0 and 100 that are divisible by 7

